Server/src/compile.sh

```diff
@@ -1,14 +1,18 @@
 # Author: Pontus Laestadius
-# Version: 1.0
+# Version: 1.2
+
+# Intro message.
+echo "-----------ERLANG COMPILATION SCRIPT VERSION 1.2-----------"
+FINISHED="-------------------------FINISHED!-------------------------"
```

Comment:
The intro message says finished?

Reply:
No you idiot learn shell scripting. I am just making a variable called FINISHED

Reply:
Popo, respecc thy coworkers